config.go

```diff
@@ -32,10 +32,15 @@ import (
 // global CPU and I/O load that logging puts on your process while attempting
 // to preserve a representative subset of your logs.
 //
-// Values configured here are per-second. See zapcore.NewSampler for details.
+// Hook is called whenever a Sampler makes a decision. Currently, whenever a
+// log is dropped.
```

Comment:
Second part no longer true. How about,

```suggestion
// If specified, the Sampler will invoke the Hook after each decision.
```